services/worker-manager/src/api.js

```diff
@@ -902,9 +902,9 @@ builder.declare({
         firstClaim: worker.firstClaim?.toJSON(),
         lastDateActive: worker.lastDateActive?.toJSON(),
         workerPoolId: worker.workerPoolId,
-        state: worker.state,
-        capacity: worker.capacity,
-        providerId: worker.providerId,
+        state: worker.state || '',
+        capacity: worker.capacity || 0,
+        providerId: worker.providerId || '',
```

Comment:
Let's have a dedicated provider for non Worker Manager workers, such as "external" or "none" or "n/a" etc.